examples/spring-boot-minimal-kotlin/build.gradle.kts

```diff
@@ -43,6 +43,10 @@ dependencies {
     configurations["kapt"].dependencies.add(project(":annotation-processor"))
 }
 
+kapt {
+    annotationProcessor("com.linecorp.armeria.server.annotation.processor.DocumentationProcessor")
+}
```

Comment:
I'm curious if we still need this even after we fixed the directory structure.

Reply:
Yes, I've tried it but it didn't invoke the annotation processor -.-

Reply:
Doh! That's a shame. No worries, then.